packages/frontend/app/routes/peers.create.tsx

```diff
@@ -144,13 +138,7 @@ export default function CreatePeerPage() {
                     error={response?.errors?.fieldErrors?.outgoingAuthToken}
                     description={
                       <>
-                        List of valid tokens to present when sending{' '}
-                        <a
-                          className='default-link'
-                          href='https://rafiki.dev/concepts/interledger-protocol/connector/#outgoing-http'
-                        >
-                          outgoing ILP packets to the peer.
-                        </a>
+                        List of valid tokens to present when sending outgoing ILP packets to the peer.
```

Comment:
@mkurapov I thought the outgoing auth token was singular, am I wrong, can it be a list as well?

Reply:
Yes, you are right. Should be singular for outgoing auth tokens :+1: